### PR TITLE
Add ci permissions to plan and dry-run

### PR DESCRIPTION
**Pull request by lewismarshall**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -61,3 +61,7 @@ variable "kops_state_bucket" {
   description = "The name of the state bucket to use for kops"
   default     = ""
 }
+
+variable "terrafrom_lock_table" {
```
Comment (KashifSaadat) [inline variables.tf]: Minor typo in variable name, `terrafrom_lock_table` to `terraform_lock_table`.

File: users.tf [hunk 2]
```diff
@@ -12,6 +12,58 @@ resource "aws_iam_group_policy_attachment" "attach_read_only_policy" {
...
+
+  policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid":      "GrantAccessToPlan",
+            "Effect":   "Allow",
+            "Action":   ["dynamodb:PutItem"],
+            "Resource": "arn:aws:dynamodb:::table/${var.terrafrom_lock_table}"
```
Comment (KashifSaadat) [inline users.tf]: Update variable name reference once corrected.

**Review by KashifSaadat — CHANGES_REQUESTED** [review]
One minor typo, otherwise LGTM.

File: variables.tf [hunk 3]
```diff
@@ -66,3 +66,7 @@ variable "kops_state_bucket" {
   description = "The name of the state bucket to use for kops"
   default     = ""
 }
+
+variable "terraform_lock_table" {
+  description = "The terraform bucket name used for state"
```
Comment (gambol99) [inline variables.tf:71]: default = ""

Comment (lewismarshall) [reply variables.tf:71]: @gambol99 all done

**Review by KashifSaadat — APPROVED** [review]
LGTM

**Review by gambol99 — APPROVED** [review]

**Pull request merged**